spec/factories/provider_user.rb

```diff
@@ -65,5 +64,11 @@
         user.provider_permissions.update_all(view_diversity_information: true)
       end
     end
+
+    trait :with_notification_preferences_enabled do
```

Comment:
Minor, but maybe just `with_notifications_enabled`?

Reply:
:+1: https://github.com/DFE-Digital/apply-for-teacher-training/pull/4879/commits/29cd7473608fe89e70e99f994918a088711d227b